### DISCUSSION TITLE
Fuction of plot  'ipfKey.orientation2color'

### DISCUSSION THREAD

**Discussion by louis96108** [mtex-toolbox/mtex#845] [Ask Anything] [answered]
I try to plot the result of Parent Beta Phase Reconstruction in Titanium Alloys,but   ‘ipfKey.orientation2color'  can not be used in MTEX 5.6.   How can I deal with it?Hoping for your reply ,thank you very much！

**Comment by ralfHielscher** [comment, marked as answer]
Could you be please a little bit more specific. What have you done, what is error message,...

Ralf.

**Reply by Savillian** [reply]
Hi all,

It seems we're both having similar issues...

For clarity, I was trying to plot a reconstructed IPF map of Beta-Ti using the following script taken from the documentation:
color = ipfKey.orientation2color(job.parentGrains.meanOrientation);
plot(job.parentGrains, color, 'figSize', 'large')

Matlab returns the following error as if the orientation2color command doesn't exist anymore. Was there a change in MTEX that wasn't transferred to the documentation yet?

**Unable to resolve the name ipfKey.orientation2color.

Error in Beta2AlphaTest560 (line 75)
color = ipfKey.orientation2color(parentEBSD('Titanium (Beta)').orientations);**

All the best,
Alec

**Reply by ralfHielscher** [reply]
The reason for this error message is that you forgot to define `ipfKey`. This is a variable name and not a function.
The documentation lists the command
``` matlab
ipfKey = ipfColorKey(ebsd('Ti (Beta)'));
```
to initialize the color key.

Ralf.

**Reply by kilir** [reply]
Hi,
how did you define ipfKey? `ipfKey = ipfHSVKey(...` or `ipfKey = ipfColorKey(...`


Cheers,
Rüdiger

**Reply by Savillian** [reply]
Hi all,

Thank you, I neglected to copy over from my previous script where I defined the ipfKey.

All the best,
Alec
